inc/classes/busting/trait-file-busting.php

```diff
@@ -130,35 +129,19 @@ private function get_all_files() {
 			return false;
 		}
 
-		$dir = $this->filesystem->dirlist( $dir_path );
-
-		if ( false === $dir ) {
-			Logger::error(
-				'Could not get the directory contents.',
-				[
-					self::LOGGER_CONTEXT,
-					'path' => $dir_path,
-				]
-			);
-			return false;
-		}
-
-		if ( ! $dir ) {
-			return [];
-		}
+		$pattern = '/' . sprintf(
+			$this->escape_file_name( $this->file_name_pattern ),
+			'([a-f0-9]{32}|local)'
+		) . '/';
 
-		$list    = [];
-		$pattern = $this->escape_file_name( $this->filename );
-		$pattern = \sprintf( $pattern, '(?<version>(?:[a-f0-9]{32}|local))' );
+		$entries = rocket_get_dir_files_by_regex( $dir_path, $pattern );
 
-		foreach ( $dir as $entry ) {
-			if ( 'f' !== $entry['type'] ) {
-				continue;
-			}
+		$list = [];
+		foreach ( $entries as $entry ) {
+			$file_name = $entry->getFilename();
```

Comment:
For commonality, let's standardize on one word for `filename`. That fits well with PHP internals too.